src/applications/personalization/dashboard-2/sass/dashboard.scss

```diff
@@ -1,3 +1,18 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
 [tabindex="-1"]:focus {
   outline: 0;
 }
+
+.dashboard {
+  .health-care {
+    hr {
+      height: 3px;
+      width: 60%;
+
+      @media (max-width: $small-screen) {
```

Comment:
Not too big of a fan of handling css this way, but it does the trick since we don't have a design utility class for anything less than `auto` or `full` width.

Reply:
This is fine, but I'd reverse it so it's mobile first. So `width: 100%` and then a `min-width` query to set the width to 60% :-D